1226 - Updates to "Using Tina Editor" for Admin changes

Enigmatical opened the pull request:
Draft of an updated "Using Tina Editor" page with placeholders for screenshots and gifs.

In a discussion with @jamespohalloran , we decided the audience for this document is:

> As a developer, I would like to know what the experience for a non-developer (editor) would be.

Additionally, we wanted to avoid referring to `local-mode` vs `production-mode`, `schema`, and `RouteMapping` plugin (mainly because they might confuse the target audience).

I will need to work with @spbyrne to get images and gifs from `tina-cloud-starter`.

Because this document is... considerably different than the previous iteration, I'm open to suggestions, constructive criticisms, and overall structure, voice, and points.

Related to #1226

Enigmatical commented on content/docs/using-tina-editor.md:
```diff
@@ -1,45 +1,43 @@
+> Gif of "Contextual Editing" with either "posts" or "pages"
 
-![using-tina-sidebar](https://res.cloudinary.com/forestry-demo/image/upload/v1638554818/tina-io/using-tina/sidebar.gif)
+However, if "Contextual Editing" is not configured or simply not available for a particular Collection, clicking on an existing Document will present the user with a fullscreen form.
```
When editing a Document that is mapped via the `RouteMapping`, we call that experience "Contextual Editing".

I'm still not sure what we should call editing a Document that isn't mapped?  "Fullscreen Editing"?  "Focused Mode"?  "Focused Editing"?

Open to suggestions here.  I wanted to avoid directly calling it "Admin Editing" (because that maintains the separation and we don't want that).

Enigmatical commented on content/docs/using-tina-editor.md:
```diff
@@ -1,45 +1,43 @@
+> Screenshot of Dashboard with Collections and Screen Plugins
 
-![tina-admin](https://res.cloudinary.com/forestry-demo/image/upload/c_scale,q_50,w_800/v1638554159/tina-io/using-tina/admin.png)
+Depending on how Tina is configured, one or more Collections will appear in the Main Navigation. Clicking on a Collection will take the user to a Document List page for that Collection.
```
I avoided briefly explaining what a Collection is here, though, I am worried about our Tina vocabulary being used without context.  Would a user, by now, know what a Collection is?

logan-anderson commented on content/docs/tinacms-context.md:
```diff
@@ -129,9 +129,72 @@ export const getStaticProps = async () => {
 
+## The `RouteMapping` Plugin
+
+The `RouteMapping` Plugin is used in the Collection List pages to route to a Document's "Contextual Editing" view rather than the default "Fullscreen Editing" view.
```
Just finding the wording of "route to" a bit confusing, could we use the word "link" or "navigate".

jamespohalloran replied:
I find "Collection List pages" also a bit confusing. Is that referring to the admin?

logan-anderson commented:
Overall looks good to me! I will give it another read once the images are in place.

spbyrne commented:
Added images. I stuck with all static images as a first pass.

Enigmatical commented on content/docs/tinacms-context.md:
```diff
@@ -129,9 +129,72 @@ export const getStaticProps = async () => {
 - `query` and `variables`: These are the same values that you would use for the [backend data-fetching](/docs/features/data-fetching/).
 - `data`: This is the production value that gets passed through to the response unchanged in production.
 
+## The `RouteMapping` Plugin
```
Same situation here as the RouteMapper plugin where I feel I'm getting mixed signals on how we want to market Tina.  On one side, I've heard from @spbyrne and @scottgallant to call the entirety of Tina (Contextual + "Admin") as Tina, so I've tried to decrease directly calling something "The Admin"... But at the same time, it appears that we don't have a good name for the location where the Main Nav takes you?

Might be worth putting our heads together on this and codifying a vocabulary.

scottgallant replied:
We should have one _Admin_, but two _editing modes_:  standard and visual.

scottgallant replied:
I feel pretty confident about that but I'm open to other opinions.

logan-anderson commented on content/docs/tinacms-context.md:
````diff
@@ -129,9 +129,72 @@ export const getStaticProps = async () => {
+This is an example of the `RouteMappingPlugin` added to our `tina-cloud-starter` template:
+
+```tsx
+import { RouteMappingPlugin } from 'tinacms';
````
This should only be loaded in edit mode. It might be best to change this example to 
```
cmsCallback={cms => {
               import('tinacms').then((package)=>{
                  cms.plugins.add(package.RouteMapperPlugin(...))
                  })
            }}
 ```
 We don't want users to increase their bundle size.

logan-anderson commented on content/docs/using-tina-editor.md:
```diff
@@ -1,45 +1,43 @@
+Tina is very flexible & extendable, so any Tina site may have a unique experience. As an introduction, we'll cover the out-of-the-box experience that most of our starters implement.
 
-## Entering Edit-mode
+## Entering the Admin
```
A bit confused on why this is "Admin" could it be something more general like "Entering the editing experience"

Enigmatical commented:
After our discussions both in this particular PR, Office Hours, and the Google Doc, I've updated a lot of the language used here.  "Contextual Editing" is now "contextual-editing", any reference to "the Admin" is now simply "the CMS", and the "Collection List" is now "Document List".

Additionally, I added a lead in to the `RouteMappingPlugin` based on @jamespohalloran 's suggestion.

Coupled with @spbyrne 's images, I'm feeling better about the narrative presented here.

logan-anderson commented:
Looks good to me! I will let someone else to the final approval since I am not confident in finding all the grammar / spelling errors.

jamespohalloran commented on content/docs/using-tina-editor.md:
```diff
@@ -1,45 +1,43 @@
+> Note: You can learn more about how to set up **contextual-editing** for your users [here](/docs/tinacms-context).
 
-![tina-markdown](https://res.cloudinary.com/forestry-demo/image/upload/c_scale,q_49,w_800/v1638554159/tina-io/using-tina/markdown.png)
+Once saved, the changes to the document, whether in contextual-editing or not, will be stored against the files.
```
Feels a tad wordy. Maybe this is fine?
```suggestion
Once saved, the changes to the document will be stored against the files.
```

Enigmatical replied:
Love it.  I was actually removing quite a bit of "wordy-ness" from this yesterday.  Good catch 👍

jamespohalloran reviewed (commented):
This looks great to me as well! Just left one little comment
